packages/webkit/SharedMemoryUnix.cpp.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/Source/WebKit/Platform/unix/SharedMemoryUnix.cpp b/Source/WebKit/Platform/unix/SharedMemoryUnix.cpp
+index aff7b0d18d..98149bc365 100644
+--- a/Source/WebKit/Platform/unix/SharedMemoryUnix.cpp
++++ b/Source/WebKit/Platform/unix/SharedMemoryUnix.cpp
+@@ -150,13 +150,13 @@ static int createSharedMemory()
+         String name = String("/WK2SharedMemory.") + String::number(static_cast<unsigned>(WTF::randomNumber() * (std::numeric_limits<unsigned>::max() + 1.0)));
+         tempName = name.utf8();
+
+-        do {
++        /*do {
+             fileDescriptor = shm_open(tempName.data(), O_CREAT | O_RDWR, S_IRUSR | S_IWUSR);
+-        } while (fileDescriptor == -1 && errno == EINTR);
++        } while (fileDescriptor == -1 && errno == EINTR);*/
+     }
+
+-    if (fileDescriptor != -1)
+-        shm_unlink(tempName.data());
++   /* if (fileDescriptor != -1)
++        shm_unlink(tempName.data());*/
+
```

Comment:
See https://github.com/termux/x11-packages/blob/master/packages/i3/posix-shm-implementation.patch for shm_open / shm_unlink example.

Reply:
Well done